frontend/class-frontend.php

```diff
@@ -346,13 +346,13 @@ public function get_default_title( $sep, $seplocation, $title = '' ) {
 		if ( ! is_string( $title ) || ( is_string( $title ) && $title === '' ) ) {
 			$title = get_bloginfo( 'name' );
```

Comment:
Boyscouting the absence of `wp_strip_all_tags` for this value, this should receive the same treatment as the other `get_bloginfo` calls in this method.

Reply:
In that case, we may as well use the `WPSEO_Utils::get_site_name()` utility function instead of duplicating the code.